pymc3_models/models/LinearRegression.py

```diff
@@ -93,17 +93,17 @@ def fit(self, X, y, inference_type='advi', minibatch_size=None, inference_args=N
 
         return self
 
-    def predict(self, X, return_std=False):
+
+    def sample(self, X, samples=2000):
```

Comment:
What do you think about calling this something like `sample_posterior`? I think it could help to be a bit more explicit.